packages/api-demo/src/app.json

```diff
@@ -4,7 +4,8 @@
         "name": "SSF Desktop Wrapper",
         "url": "http://localhost:8080/index.html",
         "uuid": "ssf-desktop-api-openfin-demo",
-        "autoShow": true
+        "autoShow": true,
+        "preload": "http://localhost:8080/scripts/containerjs-api-bundle.js"
```

Comment:
Yeah, windows opened via `window.open` and `fin.desktop.Window` inherit the preload of the parent application. For applications this isn't true, but we can pass the current applications preload script in (i.e. `fin.desktop.Window.gerCurrent().getOptions(o => preloadScriptUrl = o.preload)`) which is what i'm doing currently